graph/propertyconventions.md

```diff
@@ -0,0 +1,70 @@
+# Guidelines for the conventions of properties on graph types
+
+## Updating read-only properties
+
+### Convention
+
+It is a common call pattern for Microsoft Graph clients to `GET` an entity, make changes to a few properties, and then `PATCH` the same entity with the updated values.
+When clients do this, they often ignore that some properties are read-only or that a property may have a value that is allowed in a response, but not in a request.
+Because of this, workloads should support the practice that a request that updates a property should be considered a no-op if the value in the request matches the current value of the property.
+
```

Comment:
Can we please use our standard template https://github.com/microsoft/api-guidelines/blob/vNext/graph/patterns/PatternDescriptionTemplate.md to show what design problem we are trying to solve and examples of solving it.